### DISCUSSION TITLE
[Solved]adl.exe crashed in win10, dxgi.dll module error

### DISCUSSION THREAD

**Discussion by DJMaxIIDX** [airsdk/Adobe-Runtime-Support#816] [Q&A] [answered]
In windows10 1909 (OS Internal version 18363.959) and above, If run the code(which is below) at the beginning of the AIR for Desktop program, the application will be crash.

`import flash.display.StageDisplayState;`
`stage.displayState = StageDisplayState.FULL_SCREEN;`
`stage.nativeWindow.alwaysInFront = true;`

Steps to Reproduce:
1.Create a new File with AIR for Desktop;
2.Press Ctrl+Shift+F12 to open the publish setting panel;
3.Select AIRSDK 33.1.1.444(any version above AIR27.0) and confirm;
4.Select the first frame in timeline and press F9 to open the action panel;
5.Copy & paste the code above to the action panel;
6.Press Ctrl+Enter to test movie, and adl.exe will crash;

Windows error report log will be found in Event Viewer program. Below is the sample:

> Error Application Name: adl.exe，Version: 33.1.1.444，Timestamp: 0x6056e895
Error Module Name: dxgi.dll，Version: 10.0.19041.928，Timestamp: 0xf2d61614
Exception Code: 0xc000041d
Error Offset: 0x00031662
Error Process ID: 0x3ee8
Error Application Startup Time: 0x01d73b0df38576c8
Error Application Path: E:\Works\SDK\AIR33.1.1.444\bin\adl.exe
Error Module Path: C:\WINDOWS\system32\dxgi.dll
Report ID: 977ce9ae-f937-4c30-b6c0-ff3a23918674

Attach is windows error report file:
[Report.wer.txt](https://github.com/Gamua/Adobe-Runtime-Support/files/6381679/Report.wer.txt)

Any Workarounds:
1.Delay code execution by timer, enterframe event, or run the code after the first frame;
2.Use the AIR SDK below 27.0;

**Comment by itlancer** [comment, marked as answer]
Seems is duplicates: https://github.com/Gamua/Adobe-Runtime-Support/issues/418
I also suffering from similar issue.

**Reply by ajwfrost** [reply]
... and we should have a fix for this going into the next release. Not sure why I couldn't reproduce it when I had looked into this some time ago, but one of the other folk just hit it and can see how to work around it.

**Reply by DJMaxIIDX** [reply]
The issue has been fixed in v33.1.1.533, thank you!!
